### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by andij**

### CODE REVIEW

File: src/components/starlight/Footer.astro [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    a {
+      color: var(--sl-color-gray-3);
+    }
+
+    a:hover {
+      color: var(--sl-color-white);
+    }
+  }
```
Comment (MekalaNagarajan-Centrica) [inline src/components/starlight/Footer.astro:104]: Why is the <a> different from rest of the page?

File: src/components/starlight/EditLink.astro [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (MekalaNagarajan-Centrica) [inline src/components/starlight/EditLink.astro:1]: This logic can directly be in Footer.

File: src/components/starlight/Footer.astro (hunk 1 above)
Comment (andij) [reply src/components/starlight/Footer.astro]: There are no global link styles, there are only links styles within `.sl-markdown-content` (which the Footer is not part of.)

So I've brought the same link styles into the Footer.

**Review by RobTobias123 — COMMENTED** [review]
I like the idea of the fun element, just not quite sure what that should be yet... Toast faces seem a bit random, but maybe we could do a lighter looking set of all the masthead SVGs as a pattern in here?...

**Review by MekalaNagarajan-Centrica — APPROVED** [review]

**Pull request merged**